digital-twin-model/README.md

```diff
@@ -0,0 +1,91 @@
+
+In DTDL, a **Relationship** is a directional link from a source digital twin to a target digital twin. Relationships are used to create and navigate the graph of digital twins.
+
+Here's an example of how a relationship might be represented in a DTDL model:
+
+```json
+{
+  "@context": "dtmi:dtdl:context;3",
+  "@id": "dtmi:com:example:Building;1",
+  "@type": "Interface",
+  "displayName": "Building",
+  "contents": [
+    {
+      "@type": "Relationship",
+      "name": "hasFloor",
+      "minMultiplicity": 0,
+      "maxMultiplicity": 100,
+      "target": "dtmi:com:example:Floor;1"
+    }
+  ]
+}
+```
+
+In this example, the `hasFloor` relationship represents a link from a `Building` instance to a `Floor` instance¹. The `minMultiplicity` and `maxMultiplicity` fields specify that a `Building` can have between 0 and 100 `Floor` instances. The `target` field specifies the DTDL model that the target of the relationship is based on.
+
+And here's an example of how a relationship might be represented in a DTDL instance:
+
+```json
+{
+  "@context": "dtmi:dtdl:context;3",
+  "@id": "dtmi:com:example:Building:building1;1",  // Unique identifier for the instance
+  "@type": "dtmi:com:example:Building;1",  // The DTDL model that the instance is based on
+  "hasFloor": [
+    "dtmi:com:example:Floor:floor1;1",
+    "dtmi:com:example:Floor:floor2;1"
+  ]
+}
+```
+
+In this example, the `hasFloor` field is an array of identifiers for `Floor` instances that are related to the `Building` instance.
+
+## Using the Digital Twin Model in the Code
+
+The digital twin providers and consumers need to use the Digital Twin Model to perform digital twin interactions:
+
+- They need to use the model identifiers (denoted by "@id" in the DTDL) to identify the part of the model that they want or use.
+- They need and the model names (denoted by "name" in the DTDL model) to identify the member of a model part that they want or use.
+- They need property definitions to exchange the associated values.
+- They need commands definitions of their request and response to send and receive the appropriate payloads.
+
+We need to make this content accessible as code.  Each programming language will need its own variant.  In this code repository, we will
+provide a Rust variant. We will place all of the code for the model's content in a single file named after the model and its version. In
+this code repository, that file will be the 'digital-twin-model/src/sdv_v1.rs' file.
+
+The 'sdv_v1.rs' file is based on the model content from the DTDL files located under 'digital-twin-model/dtdl/dtmi/sdv'.
```

Comment:
we should probably also document what sdv_v0.rs is. I don't think we need to say anything more than "it's an earlier prototype for legacy support" or something like that

Reply:
Fixed.